src/main.java

```diff
@@ -1,5 +1,77 @@
 public class main {
     public static void main(String[] args){
+        // Задача 1
+        byte aa = 1;
+        short bb = 2;
+        int cc = 3;
+        long dd = 4L;
+        float ee = 1.1f;
+        double ff = 1.1;
+        char gg = 'a';
+        boolean hh = true;
 
+        // Задача 2
+        float weightBoxer1 = 78.2f;
+        float weightBoxer2 = 82.7f;
+        float weightAllBoxers = weightBoxer1 + weightBoxer2;
+        float weightDifference = weightBoxer2 - weightBoxer1;
+        System.out.println("Вес боксёров " + weightAllBoxers + " кг");
+        System.out.println("Разница в весе " + weightDifference + " кг");
+
+        // Задача 3
+        byte weightBanana = 80;
+        byte weightCupOfMilk = 105;
+        byte weightIceCream = 100;
+        byte weightEgg = 70;
+        float weightBreakfastInKg = (5 * weightBanana + 2 * weightCupOfMilk + 2 * weightIceCream + 4 * weightEgg) / 1000f;
+        System.out.println("Вес завтрака " + weightBreakfastInKg + " кг");
+
+        // Задача 4
+        short fullWeightLoss = 7;
+        short minWeightLoss = 250;
+        short maxWeightLoss = 500;
+        float maxDaysOfWeightLoss = fullWeightLoss * 1000f / minWeightLoss;
+        float minDaysOfWeightLoss = fullWeightLoss * 1000f / maxWeightLoss;
+        float averageDaysOfWeightLoss = (maxDaysOfWeightLoss + minDaysOfWeightLoss) / 2f;
+        System.out.println("Максимальное время похудения " + maxDaysOfWeightLoss + " дней");
+        System.out.println("Минимальной время похудения " + minDaysOfWeightLoss + " дней");
+        System.out.println("Среднее время похудения " + averageDaysOfWeightLoss + " дней");
+
+        // Задача 5
+        int mashaSalary1 = 67_760;
+        int denisSalary1 = 83_690;
+        int kristinaSalary1 = 76_230;
+        int mashaIncrease = mashaSalary1 / 10;
+        int mashaSalary2 = mashaSalary1 + mashaIncrease;
+        int denisIncrease = denisSalary1 / 10;
+        int denisSalary2 = denisSalary1 + denisIncrease;
+        int kristinaIncrease = kristinaSalary1 / 10;
+        int kristinaSalary2 = kristinaSalary1 + kristinaIncrease;
+        System.out.println("Маша теперь получает " + mashaSalary2 + " рублей. Годовой доход вырос на " + mashaIncrease + " рублей");
+        System.out.println("Денис теперь получает " + denisSalary2 + " рублей. Годовой доход вырос на " + denisIncrease + " рублей");
+        System.out.println("Кристина теперь получает " + kristinaSalary2 + " рублей. Годовой доход вырос на " + kristinaIncrease + " рублей");
+
+        // Задача 6
+        int a = 12;
+        int b = 27;
+        int c = 44;
+        int d = 15;
+        int e = 9;
+        int result = a * (b + (c - d * e));
+        result = - result;
```

Comment:
еще один возможный вариант:
```suggestion
        result *= -1;
```